pipelines/organisations/schema.json

```diff
@@ -0,0 +1,69 @@
+{
+    "@context": [
+      "http://www.w3.org/ns/csvw",
+      {
+        "@language": "en"
+      }
+    ],
+    "tables": [
+      {
+          "url": "organisations.csv",
+          "tableSchema": {
+              "columns": [
+                  {
+                      "name": "organisation",
+                      "titles": "organisation",
+                      "datatype": "string",
+                      "suppressOutput": true
```

Comment:
Do we really want to suppress output of the organisation's URI? Surely the `organisation` column is effectively the same as the `hasRegisteredSite` column? Should we just remove it if we're not using it?